client/transactions/list/index.tsx

```diff
@@ -583,10 +583,9 @@ export const TransactionsList = (
 	const downloadable = !! rows.length;
 
 	const onDownload = async () => {
-		recordEvent( 'wcpay_transactions_download_csv_click', {
+		recordEvent( 'wcpay_csv_export_click', {
 			location: props.depositId ? 'deposit_details' : 'transactions',
-			exported_transactions: rows.length,
```

Comment:
I think we should include a prop specifically for the data "type" / entity being exported (e.g. `row_type` or similar – payouts, transactions, disputes etc). We can infer from `location`/`page`, but it's not the same thing. Also if we export the same data type from different screens (e.g. `row_type`=`transactions`), then we can get usage stats for that data type, across both screens (if we want). More flexible.

Reply:
Added an explicit `type` prop here - 51f35a6